src/applications/check-in/components/UpcomingAppointmentsList.jsx

```diff
@@ -86,19 +86,28 @@ const UpcomingAppointmentsList = props => {
                         {t('date-day-of-week', { date: dayStartTime })}
                       </h4>
                     </div>
-                    <div className="vads-l-col--10 vads-u-border-top--1px">
+                    <div className="vads-l-col--10 vads-u-border-top--1px vads-u-border-color--gray-light">
                       <ul
-                        className="vads-u-margin-bottom--4 check-in--appointment-list appointment-list"
+                        className="vads-u-margin-bottom--3 check-in--appointment-list appointment-list"
                         data-testid="appointment-list"
                       >
-                        {appointments.map(appointment => {
+                        {appointments.map((appointment, number) => {
                           return (
-                            <UpcomingAppointmentsListItem
-                              key={getAppointmentId(appointment)}
-                              appointment={appointment}
-                              goToDetails={handleDetailClick}
-                              router={router}
-                            />
+                            <div
```

Comment:
this DIV should go inside the <li> the immediate child of a <ul> should be an <li>